Dockerfile

```diff
@@ -1,6 +1,9 @@
 ARG registry
 FROM ${registry}docker.io/library/python:3.9.17-bullseye
 
+RUN apt-get update \
```

Comment:
Just like with the other images, we need a way to force this layer to be remade, instead of a cached layer being reused. This interacts with `azul_docker_image_cache_seed` and `azul_docker_image_tag`. Please request PL on this.